### ISSUE TITLE
Windows configuration / PATH

### ISSUE THREAD

**Issue by hustf** [julia-vscode/julia-vscode#475]
The Julia executable is on my path, but I get this:
`Could not start the julia language server. Make sure the configuration setting julia.executablePath points to the julia binary.`

PATH looks like this from within Julia. I would say it points to the default location for most users. 
```
julia> ENV["PATH"]
"c:\\Users\\F\\AppData\\Local\\Julia-0.6.2\\bin;C:\\Progr....
 ```

The fix, like recommended, still took some work to get right:
![image](https://user-images.githubusercontent.com/13949620/36948763-033f7fea-1fe0-11e8-8341-e28db85ceaff.png)

**Comment by davidanthoff** [comment]
Hm, not sure what is going on there... But, the next version revamps that whole part of the extension, so maybe best if we just see whether this is still a problem in the next version.

**Comment by djm158** [comment]
I'm seeing this as well 
![image](https://user-images.githubusercontent.com/8800738/38311736-ee693770-37ed-11e8-83fc-1803392e0c0a.png)

```
julia> ENV["PATH"]
"C:\\Julia\\Julia-0.6.2\\bin;C:\\...
```

**Comment by KristofferC** [comment]
Do you need to add `julia.exe` in the end?

**Comment by djm158** [comment]
ah that seemed to do the trick. thanks!

**Comment by jaubourg** [comment]
I'm on windows 10.

`julia.exe` is in `C:\Users\Julian\AppData\Local\Julia-0.6.2\bin` which is itself is in the PATH. Yet the extension won't find the exe.

So I set `julia.executablePath` to `"C:\\Users\\Julian\\AppData\\Local\\Julia-0.6.2\\bin\\julia.exe". No more `"exe not found"`  or whatever message (at least after a restart of vscode) but when I tried `toggle linter` it said the `server is not yet running`.

Now it seems to work and I see a julia process in the vscode group in the task manager.

I think the extension is in serious need of some clear notifications for when the julia server is being started and when it actually is up.

Also, the fact it cannot find the exe when it's in the PATH is a bit of an issue.
